runtime/queries/julia/injections.scm

```diff
@@ -1,7 +1,6 @@
-; TODO: re-add when markdown is added.
-; ((triple_string) @injection.content
-;  (#offset! @injection.content 0 3 0 -3)
-;  (#set! injection.language "markdown"))
+((triple_string) @injection.content
+ (#offset! @injection.content 0 3 0 -3)
```

Comment:
```suggestion
```

`#offset` is not implemented in helix (this was probably copied over from the neovim queries, and neovim does implement `#offset`)

Reply:
This is not fixed so I unresolved it.

Reply:
sorry about that ;)